### PR TITLE
GridFilesystemCacheProvider, MavenModelProcessor, other minor improvements

### PR DESCRIPTION
**Pull request by jdcasey**
Also, trim down the method calls supported by Transfer in order to support non-file based cache providers, and introduce a CacheProvider.AdminView to support delegating cache providers to one another

### CODE REVIEW

**Comment by jdcasey** [comment]
retest this please

**Comment by jdcasey** [comment]
retest this please

File: pom.xml [hunk 1]
```diff
@@ -49,9 +49,10 @@
     <atlasVersion>0.16.0</atlasVersion>
     <partylineVersion>1.5</partylineVersion>
     <jhttpcVersion>1.1</jhttpcVersion>
+    <infinispanVersion>8.1.2.Final</infinispanVersion>
     <!-- 1.6 is necessary for certain subsets, so it can be used by eg. PME -->
     <!-- 1.7 is the minimum JVM we can use to build, because of deps in other parts. -->
-    <javaVersion>1.7</javaVersion>
+    <javaVersion>1.8</javaVersion>
   </properties>
```
Comment (rnc) [inline pom.xml]: How will this work with PME baseline requirements?

Comment (jdcasey) [inline pom.xml]: @rnc The compiler source/target should still be set separately, I thought, but I'll check.

**Pull request merged**